tests/service/generate_test_data.yml

```diff
@@ -0,0 +1,98 @@
+# Generate lists for hosts and services
+---
+- name: Get Domain from server name
+  ansible.builtin.set_fact:
+    ipaserver_domain: "{{ ansible_facts['fqdn'].split('.')[1:] | join ('.') }}"
```

Comment:
ansible-lint is warning of a white space after `join`. It would be nice to remove the white space.